Add guidance for features enabled by cloudlink (#3219)
balena-io/docs: pages/learn/welcome/security.md

### Cloudlink

Cloudlink status in the dashboard represents a device connection to the balenaCloud backend. This connection is the underlying technology that enables the following functionalities within balenaCloud, and as such we indicate the status of Cloudlink to help you to understand the status of your device and what functionality may or may not be available.

- [SSH access](/learn/manage/ssh-access/) via CLI or web terminal
- [Public Device Url](/learn/manage/actions/#public-device-url)
- [Device actions](/learn/manage/actions/#device-actions) that allow controlling device state
- [Host OS Updates](/reference/OS/updates/self-service/)
- [Device Diagnostics](/reference/diagnostics/)

> Currently, Cloudlink uses [OpenVPN](https://openvpn.net/) as an underlying technology to achieve these things but this is subject to change with better technology, this is why we abstract away from the technology and call the component as Cloudlink. Devices only connect outbound to the Cloudlink and all traffic over Cloudlink is encrypted with TLS.

Reviewer: ```suggestion
Currently, Cloudlink uses [OpenVPN](https://openvpn.net/) as an underlying technology to achieve its functionality. Devices only connect outbound to the Cloudlink, and all traffic over Cloudlink is encrypted with TLS.
```
